business/definitionService.js

```diff
@@ -159,8 +159,6 @@ class DefinitionService {
         })
       )
     )
-    // don't forget to store anything left over
-    return this.search.store(indexes)
```

Comment:
I've been trying to diagnose what's going on here but haven't had much luck. `indexes` isn't declared, so this didn't work anyway. I had hacked around that today by storing `list` instead, which seemed to work. Removing this entirely leaves me with an empty index (at least for in-memory + file stores) and a bunch of errors looking for yaml files that don't exist, e.g. `'/tmp/cd-5G4p2p/curated-data-dev/curations/npm/npmjs/-/colors/.yaml'`